src/ui/components/ui/balance/Balance.module.css

```diff
@@ -0,0 +1,10 @@
+.usdAmount {
+  color: #9ba6b2;
```

Comment:
We have defined css color variables at `app.styl`, so replace it with `color: var(--color-basic500)`